### DISCUSSION TITLE
Access Logs and Whitelisted Access

### DISCUSSION THREAD

**Discussion by rsidhaarth** [Blair2004/NexoPOS#453] [Ideas]
Hi Team,

It will be more helpful if you provide access logs and so we can find these details

1) user account
2) browser used
3) os used
4) IP logged in

And also, we can ban IP and also we can whitelist IP.

Also, we can allow users to log in only on their whitelisted IP and IP can be whitelisted by wildcard as well

1.1.1.1

or 1.1.1.*

or 1.1.*.*

These option will be more helpful as it is self hosted and 100% chance people can share the details with other people.

**Comment by Blair2004** [comment]
Hi, nice one.
For such type of suggestion that is not always needed by owner, we'll rather make it available as a module.
Suggestion taken in consideration.

Kind Regards.
